packages/cli-kit/src/node/error-handler.ts

```diff
@@ -26,15 +27,17 @@ export function errorHandler(error: Error & {exitCode?: number | undefined}) {
       .then((error: Error) => {
         return handler(error)
       })
-      .then(reportError)
+      .then((mappedError) => reportError(mappedError, config))
       .then(() => {
         process.exit(1)
       })
   }
 }
 
-const reportError = async (error: Error): Promise<Error> => {
-  await reportEvent({errorMessage: error.message})
+const reportError = async (error: Error, config?: Interfaces.Config): Promise<Error> => {
+  if (config !== undefined) {
+    await reportEvent({config, errorMessage: error.message})
+  }
   if (settings.debug || !shouldReportError(error)) return error
```

Comment:
The `runCLI` uses `catch`, and connects it to `errorHandler`, which connects to this, but there's no command config available at that point, as it's happening outside of the scope of a command. I don't really want to _drop_ the `catch` because there may be some benefit or reason to it and it does still hit Bugsnag.